beluga/CMakeLists.txt

```diff
@@ -48,6 +48,13 @@ find_package(range-v3 REQUIRED)
 find_package(Sophus REQUIRED)
 find_package(TBB REQUIRED)
 
+list(APPEND CMAKE_MODULE_PATH "/usr/lib/x86_64-linux-gnu/cmake/OpenVDB/")
+list(APPEND CMAKE_MODULE_PATH "/usr/lib/aarch64-linux-gnu/cmake/OpenVDB")
+find_package(OpenVDB QUIET)
+if(OpenVDB_FOUND)
+  add_definitions(-USE_OPENVDB)
```

Comment:
@pvela2017 consider using `BELUGA_OPENVDB_SUPPORT` for a compile definition. Also, use `target_compile_definitions` for the `${PROJECT_NAME}` target instead of a blanket `add_definitions` macro.

Reply:
Updated accordingly in bef40ad2dd85beba55969b16c62adab2f0f94e79